Marlin/ubl_motion.cpp

```diff
@@ -31,6 +31,12 @@
   #include <math.h>
 
   extern float destination[XYZE];
+  #if ENABLED(UBL_G26_MESH_VALIDATION)
+    extern bool g26_debug_flag;
+  #else
+    bool g26_debug_flag=false;
```

Comment:
`constexpr bool g26_debug_flag = false;`?

Reply:
The UBL_motion.cpp file still uses g26_debug_flag to spit out extra information about the mesh cells and where the nozzle is passing.    So in a sense, it is still slightly UBL related.   That is why it needs a place to be defined even if G26 is turned off.

This is a temporary work around...  Maybe it should be called detailed_movement_debug_flag, or something like that.   That would make it more natural to find a home for it.   And maybe leverage in other movement functions...